include/deal.II/fe/fe_base.h

```diff
@@ -567,13 +590,13 @@ FiniteElementData<dim>::n_dofs_per_object() const
   switch (structdim)
     {
       case 0:
-        return dofs_per_vertex;
+        return dofs_per_vertex + global_dofs_per_cell;
       case 1:
-        return dofs_per_line;
+        return dofs_per_line + global_dofs_per_cell;
```

Comment:
given the way we calculate `dofs_per_cell` below in the constructor, I would say that we don't need to touch this at all, i.e. don't add `global_dofs_per_cell` here.

But before doing this changes, let's wait for other devs to have a look.